spec/Section 2 -- Language.md

```diff
@@ -1203,6 +1203,10 @@ As future versions of GraphQL adopt new configurable execution capabilities,
 they may be exposed via directives. GraphQL services and tools may also provide
 any additional *custom directive* beyond those described here.
 
+Note: Directives simply provide additional information. The normative force
+of directives may vary and may also depend on a directive's arguments or
+location, as well as the context in which the directive is encountered.
+
```

Comment:
Except for those spec'd by this document? Those should be normative right?

Reply:
tried to clarify that all directives MUST be respected, but that they may prescribe behavior changes that are OPTIONAL